sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/schema/BeamSqlRecordCoder.java

```diff
@@ -34,13 +35,12 @@
 import org.apache.beam.sdk.coders.InstantCoder;
 import org.apache.beam.sdk.coders.ListCoder;
 import org.apache.beam.sdk.coders.StringUtf8Coder;
-import org.apache.beam.sdk.extensions.sql.impl.utils.CalciteUtils;
 
 /**
- *  A {@link Coder} encodes {@link BeamSqlRow}.
+ *  A {@link Coder} encodes {@link BeamSqlRecord}.
  */
-public class BeamSqlRowCoder extends CustomCoder<BeamSqlRow> {
-  private BeamSqlRowType tableSchema;
+public class BeamSqlRecordCoder extends CustomCoder<BeamSqlRecord> {
```

Comment:
BTW, BeamRecordType would take this list of coders in its constructor so that BeamRecordCoder could be created given a BeamRecordType.